lang/main.json

```diff
@@ -296,7 +293,6 @@
         "recordingToken": "Enter recording token",
         "passwordCheck": "Are you sure you would like to remove your password?",
         "passwordMsg": "Set a password to lock your room",
-        "shareLink": "Share the link to the call",
```

Comment:
"passwordCheck" and "passwordMsg" and "yourPassword" aren't used anymore it appears. Not sure if they were even used in the invite, but if you confirm can you remove them anyway.